docs/user-guide/test-zen-doc_Sean.md

```diff
@@ -53,50 +52,72 @@ FTP Port  |The FTP Port number for internal use. The default port is 21. If not
+|Certificates Option 2|Zowe will create a keyring and populate the keyring by connecting pre-existing certificates and CAs that you specify.|RACF: [ZWEIKRR2](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIKRR2)<br /><br />TSS: [ZWEIKRT2](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIKRT2)<br /><br />ACF2: [ZWEIKRA2](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIKRA2)|
+|Certificates Option 3|Zowe will create a keyring and populate the keyring by importing PKCS12 content from a dataset that you specify.|RACF: [ZWEIKRR3](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIKRR3)<br /><br />TSS: [ZWEIKRT3](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIKRT3)<br /><br />ACF2: [ZWEIKRA3](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIKRA3)|
+|STC (NYI)|**Purpose**:<br /> ZWESLSTC is the job for running Zowe's webservers. ZWESISTC is for running the APF authorized cross-memory server. The ZWESASTC job is started by ZWESISTC on an as-needed basis.<br /><br />**Action**:<br /> Copy the members ZWESLSTC, ZWESISTC, and ZWESASTC into your desired PROCLIB. If the job names are customized, modify the job name YAML values in `zowe.setup.security.stcs`|[ZWEISTC](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEISTC)|
+|(Optional) VSAM for Caching Service (NYI)|**Action:**<br /> Create a RLM or NONRLM dataset for the Caching service, and set the name into the YAML value `components.caching-service.storage.vsam.name`|[ZWECSVSM](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWECSVSM)|
 
-* View/Edit YAML: This button lets you preview or adjust the YAML configuration that is used by Zowe. The prompts of the wizard are used to automatically generate its contents, but you can review it, edit it, or import and export its contents.
-* View/Submit Job: When you are happy with the values for the step, you can press the job submit button to execute the operations of the configuration step upon the z/OS system.
-* View Job output: Upon submission of the job, the output will be available for review.
 
 ## Final Review
 
-The last page of the wizard contains a summary of which steps were completed, skipped, or had errors. You can revisit any step to retry it, and you can also use this last page for exporting the final YAML that was generated.
- 
+After completing the steps presented in the wizard, a summary is provided indicating which steps were completed, skipped, or have errors. You can revisit any step to retry performing the step. You also have the option to export the final generated YAML file.
```

Comment:
"Errors are stored in the View Job Output. You can revisit any step to retry performing the step."